core/src/main/scala/org/apache/spark/ExecutorAllocationClient.scala

```diff
@@ -55,18 +55,18 @@ private[spark] trait ExecutorAllocationClient {
   /**
    * Request that the cluster manager kill the specified executors.
    *
-   * When asking the executor to be replaced, the executor loss is considered a failure, and
-   * killed tasks that are running on the executor will count towards the failure limits. If no
-   * replacement is being requested, then the tasks will not count towards the limit.
-   *
    * @param executorIds identifiers of executors to kill
-   * @param replace whether to replace the killed executors with new ones, default false
+   * @param adjustTargetNumExecutors whether the target number of executors will be adjusted down
+   *                                 after these executors have been killed
+   * @param countFailures if there are tasks running on the executors when they are killed, whether
```

Comment:
I'm still a little confused about this parameter.

If `force = false`, it's a no op. And all call sites I've seen seem to set this parameter to `false`. So is there something I'm missing?

Reply:
whoops, I was supposed to set `countFailures = true` in `sc.killAndReplaceExecutors`, thanks for catching that.